source/sass/image-picker.scss

```diff
@@ -22,25 +22,32 @@ ul.thumbnails.image_picker_selector
     margin:               0px;
   }
 
+  li.group
+  {
+    width: 100%
+  }
+
   li.group_title
   {
     float: none;
   }
 
   li
   {
-    margin:   0px $thumbnail_margin $thumbnail_margin 0px;
+    margin:   4px $thumbnail_margin $thumbnail_margin 4px;
     float:    left;
 
     .thumbnail
     {
       padding:  $thumbnail_padding;
       border:   $thumbnail_border;
+      border-radius: 4px;
```

Comment:
I'd prefer to keep this very minimal so each user styles this as needed. Is this change for any functionality or just because it looks better?

Reply:
This can probably be removed

Reply:
Border-radius isn't really necessary.  I did it originally to match the radius of the focus outline.

Reply:
I should also mention that the addition of the 4px margin is needed in order to see the entire focus ring around an element.

Reply:
I say we should remove this for now, let the users modify as needed